### PR TITLE
Add aimec plots to report and move atomic funcs to aimectools

### LINKED ISSUE #354: separate atomic functions from ana3Aimec to aimecTools

### PR DESCRIPTION
**Pull request by awirb**
* update aimec ini files 
* now print reference and comparison module name in area comparison plot 
* move atomic aimec functions to aimecTools 
* make runDamBreak run again - however see issue #334 regarding results 
* upate Inclined Path aimec
*  fix #354

### CODE REVIEW

File: avaframe/ana3AIMEC/aimecTools.py [hunk 1]
```diff
@@ -0,0 +1,933 @@
...
+
+    for i in range(nTopo):
+        rasterdata = dataPressure[i]
+
+        """
+        area
+        # true positive: reality(mask)=1, model(rasterdata)=1
+        # false negative: reality(mask)=1, model(rasterdata)=0
+        # false positive: reality(mask)=0, model(rasterdata)=1
+        # true negative: reality(mask)=0, model(rasterdata)=0
```
Comment (fso42) [inline avaframe/ana3AIMEC/aimecTools.py:787]: Mh, maybe change reality to result1 and model to result2 ?

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/ana3AIMEC/aimecTools.py [hunk 2]
```diff
@@ -0,0 +1,933 @@
...
+
+    demSource = glob.glob(os.path.join(avalancheDir, 'Inputs', '*.asc'))
+    try:
+        assert len(demSource) == 1, 'There should be exactly one topography .asc file in ' + \
+            avalancheDir + '/Inputs/'
+    except AssertionError:
+        raise
+    cfgPath['demSource'] = ''.join(demSource)
+
+    pathResult = os.path.join(avalancheDir, 'Outputs', 'ana3AIMEC', dirName)
```
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/ana3AIMEC/aimecTools.py:74]: the problem is that this should also work with simName, as for the comparison of simulations from two modules, this directory is called like the simulation name as it does not belong to one single module

File: avaframe/ana3AIMEC/aimecTools.py [hunk 3]
```diff
@@ -0,0 +1,933 @@
...
+    # output: Left and right side points for the domain
+    rasterTransfo = geoTrans.path2domain(avaPath, rasterTransfo)
+
+    # Make transformation matrix
+    rasterTransfo = makeTransfoMat(rasterTransfo)
+
+    # calculate the real area of the new cells as well as the scoord
+    rasterTransfo = getSArea(rasterTransfo)
+
+    log.info('Size of rasterdata- old: %d x %d - new: %d x %d' % (
```
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/ana3AIMEC/aimecTools.py:172]: done

File: avaframe/ana3AIMEC/aimecTools.py [hunk 4]
```diff
@@ -0,0 +1,933 @@
...
+    x2 = xcoord[1:n, 0:m-1]
+    y2 = ycoord[1:n, 0:m-1]
+    # i = 3 : bottom right in the matrix : [1:n, 1:m]
+    x3 = xcoord[1:n, 1:m]
+    y3 = ycoord[1:n, 1:m]
+    # i = 4 : bottom left in the matrix : [0:n-1, 1:m]
+    x4 = xcoord[0:n-1, 1:m]
+    y4 = ycoord[0:n-1, 1:m]
+
+    Area = np.abs(x1*y2-y1*x2 + x2*y3-y2*x3 + x3*y4-y3*x4 + x4*y1-y4*x1)/2
```
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/ana3AIMEC/aimecTools.py:421]: done

File: avaframe/ana3AIMEC/aimecTools.py [hunk 5]
```diff
@@ -0,0 +1,933 @@
...
+
+    # Method 2
+    # calculate area of each cell assuming they are parallelogramms
+    # (which is wrong)
+    # newAreaRaster = np.abs(dxl*dys - dxs*dyl)
+
+    # save Area matrix
+    rasterTransfo['rasterArea'] = Area
```
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/ana3AIMEC/aimecTools.py:429]: d

File: avaframe/ana3AIMEC/aimecTools.py [hunk 6]
```diff
@@ -0,0 +1,933 @@
...
+                index for start of the runout area (in s)
+    """
+    # Read input parameters
+    demSource = cfgPath['demSource']
+    ProfileLayer = cfgPath['profileLayer']
+    splitPointSource = cfgPath['splitPointSource']
+    DefaultName = cfgPath['projectName']
+
+    w = float(cfgSetup['domainWidth'])
+    startOfRunoutAngle = float(cfgSetup['startOfRunoutAngle'])
```
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/ana3AIMEC/aimecTools.py:132]: Done

File: avaframe/ana3AIMEC/aimecTools.py [hunk 7]
```diff
@@ -0,0 +1,933 @@
...
+        assert len(profileLayer) == 1, message
+    except AssertionError:
+        raise
+    cfgPath['profileLayer'] = ''.join(profileLayer)
+
+    splitPointLayer = glob.glob(os.path.join(avalancheDir, 'Inputs', 'POINTS',
+                                             '*.shp'))
+    try:
+        message = 'There should be exactly one .shp file containing the split point in ' + avalancheDir + '/Inputs/POINTS/'
+        assert len(splitPointLayer) == 1, message
```
[the remark this answers is not in the document]
Comment (matthiasto) [reply avaframe/ana3AIMEC/aimecTools.py:61]: Yes in alpha beta multiple points within the same shp file are allowed but once you give a path, only the closest point to this path will be retained. 
We can make a similar point file for aimec: splitPointAIMEC

File: avaframe/ana3AIMEC/aimecTools.py [hunk 8]
```diff
@@ -0,0 +1,933 @@
...
+    # calculate the real area of the new cells as well as the scoord
+    rasterTransfo = getSArea(rasterTransfo)
+
+    log.info('Size of rasterdata- old: %d x %d - new: %d x %d' % (
+        np.size(rasterdata, 0), np.size(rasterdata, 1),
+        np.size(rasterTransfo['gridx'], 0),
+        np.size(rasterTransfo['gridx'], 1)))
+
+    ##########################################################################
+    # affect values
```
[the remark this answers is not in the document]
Comment (matthiasto) [reply avaframe/ana3AIMEC/aimecTools.py:178]: nothing, this can be removed

File: avaframe/ana3AIMEC/aimecTools.py (hunk 1 above)
Comment (matthiasto) [reply avaframe/ana3AIMEC/aimecTools.py:787]: :+1:

**Review by matthiasto — APPROVED** [review]

**Review by fso42 — APPROVED** [review]

File: avaframe/ana3AIMEC/aimecTools.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (fso42) [reply avaframe/ana3AIMEC/aimecTools.py:61]: So to be clear: we still need the split point for AIMEC?

File: avaframe/ana3AIMEC/aimecTools.py (hunk 8 above)
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/ana3AIMEC/aimecTools.py:178]: removed the # affect values

File: avaframe/ana3AIMEC/aimecTools.py (hunk 1 above)
Comment (awirb) [reply avaframe/ana3AIMEC/aimecTools.py:787]: done

**Pull request merged**